src/form/Calendar/CalendarDays/CalendarDays.tsx

```diff
@@ -74,50 +100,64 @@ const ZERO_TIME = {
   milliseconds: 0
 };
 
+const DAY_OF_WEEK_LABELS = ['Su', 'Mo', 'Tu', 'We', 'Th', 'Fr', 'Sa'];
```

Comment:
maybe need to allow for customization in the future?

Reply:
Something like this - https://ionel-lupu.com/blog/how-to-display-weekday-names-in-your-users-language